Makefile

```
@@ -43,7 +57,7 @@ info:  ## Show info about env and used variables
 	@echo "DOCKER_PASSWORD=NOT SHOWN"
 
 code-quality:  ## Run code quality tools
-	cd "${CODE_QUALITY_DIR}" && make all
+	make black flake  # mypy disabled for testing
```

I think this will produce additional line of the output like `make black flake`.

```suggestion
code-quality: black flake  ## Run code quality tools
```
OR
```suggestion
code-quality:  ## Run code quality tools
	@make black flake  
```